packages/react-composites/src/composites/ChatComposite/file-sharing/FileUploadButton.tsx

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { CommunicationIdentifierKind } from '@azure/communication-common';
+import { Icon, mergeStyles, Stack, useTheme } from '@fluentui/react';
+import React, { useRef } from 'react';
+import { FileUploadHandler } from './FileUploadHandler';
+import { UploadedFile } from './UploadedFile';
+
+/**
+ * Props for {@link FileUploadButton} component.
+ * @internal
+ */
+export interface FileUploadButtonProps {
+  /**
+   * The ACS user ID of the user uploading the file.
+   */
+  userId: CommunicationIdentifierKind;
+  /**
+   * The function of type {@link FileUploadHandler} for handling file uploads.
+   */
+  fileUploadHandler?: FileUploadHandler;
```

Comment:
You can define and use `FileUploadHandler` internally just fine. You'd conditionally export it through the public API only in beta flavor.

Reply:
Didn't get you